src/components/ServiceContainers/ServiceContainerLoading.tsx

```diff
@@ -62,7 +62,7 @@ export const ServiceContainerLoading = ({
   //  - request was successfully finished,
   //  - content property is available (= content property means table data with pagination are expected),
   //  - but no items are available
-  if (data.content && !data.content.length) return <EmptyStateCard title={title} />;
+  if (data.content && !data.content.length && !isInline) return <EmptyStateCard title={title} />;
```

Comment:
if `isInline` is true, then something else needs to be displayed, for example html entity `&ndash;` can be tested

Reply:
For now, we decided to keep this condition -- children will be displayed when data are empty.